tests/sentry/grouping/snapshots/test_grouphash_metadata/test_metadata_from_variants/newstyle@2023_01_11/unreal_assertion_check_fail_on_windows.pysnap

```diff
@@ -18,108 +18,209 @@ metrics with tags: {
   "grouping.grouphashmetadata.event_hashing_metadata.stacktrace": {
     "chained_exception": "False",
     "stacktrace_location": "exception",
-    "stacktrace_type": "system"
+    "stacktrace_type": "in_app"
   }
 }
 ---
 contributing variants:
+  app*
+    hash: "21f12614b98d6dd2c5d5303e452b4c6b"
+    contributing component: exception
+    component:
+      app*
+        exception*
+          stacktrace*
+            frame* (marked in-app by stack trace rule (family:native function:FDebug::CheckVerifyFailedImpl* v+app))
+              function*
+                "WinMain"
+            frame* (marked in-app by stack trace rule (family:native function:FDebug::CheckVerifyFailedImpl* v+app))
+              function*
+                "LaunchWindowsStartup"
```

Comment:
Same with these. We might be able to handle some of this by adding a `category:system -app` rule, but that should probably be a separate PR so we can isolate and audit the effects of that change.